deploy/manifests/99_openshift-machineconfig_99-additional-master-ssh.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  creationTimestamp: null
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-additional-master-ssh
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+    passwd:
+      users:
+      - name: core
+        sshAuthorizedKeys:
+        - REPLACE_SSH_KEY
+  fips: false
```

Comment:
Why include this and the other fields below? Can we remove these so we're only setting the ssh key?

Reply:
We can, removing.